dashboard/app/assets/stylesheets/util.scss

```diff
@@ -0,0 +1,4 @@
+@mixin font-main($color, $weight) {
```

Comment:
@mDibyo I do like mixins, but let's only use them when they do tricky things like calculations.
Could we remove this one?